### PR TITLE
Expose on:change event handler on TextInput

### PR DESCRIPTION
**Pull request by ehhong**

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viamrobotics/prime-core",
-  "version": "0.0.99",
+  "version": "0.0.991",
```
Comment (ehhong) [inline packages/core/package.json:3]: how do I upgrade the prime version after `0.0.99`?

Comment (mrloureed) [reply packages/core/package.json:3]: `0.0.100` I would think

Comment (ehhong) [reply packages/core/package.json:3]: but isn't `0.0.100` < `0.0.99`?

Comment (DTCurrie) [reply packages/core/package.json:3]: 0.0.100 is correct.

We will hit 0.1.0 someday.

**Review by DTCurrie — APPROVED** [review]
✅ pending version change.

**Review by ethanlook — APPROVED** [review]

**Pull request merged**